tests/kuttl/common/assert-sample-deployment.yaml

```diff
@@ -9,3 +9,19 @@ spec:
     SESSION_TIMEOUT = 3600
 status:
   readyCount: 1
+---
+# Test the status code is correct for each endpoint
+# This test is for heat endpoints
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+commands:
+  - script: |
+      set -x
+      RETURN_CODE=0
+      PUBLIC_URL=$(oc get route -n $NAMESPACE horizon-public -o jsonpath='{.status.ingress[0].host}')
```

Comment:
I think we can use Status.Endpoint of Horizon CR instead.

Reply:
Yeah, we could do that. I just copied this from Heat. But we can do it either way I guess.

Reply:
Ok, updated to use the `status.endpoint`